.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+
+name: Build
+
+on:
+  push:
+    branches: [main, dev]
+  pull_request:
+    branches: [main, dev]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 17.x
```

Comment:
The node version, is not the same in the 3 workflows (**17** and **16**). Besides you shouldn't use an odd version number (17) [which are unstable](https://github.com/nodejs/release#release-phases)

Reply:
Fixed: https://github.com/aleph-im/aleph-sdk-ts/pull/49/commits/ddff75a4e3970ccaa4a4f5b4d115b8a5ad7f8c82